yt_dlp/extractor/cbc.py

```diff
@@ -7,16 +7,21 @@
 
 from .common import InfoExtractor
 from ..utils import (
+    base_url,
     ExtractorError,
```

Comment:
Uppercase comes first
```suggestion
    ExtractorError,
    base_url,
```